src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -208,6 +208,9 @@ public static function get_theme_data( $deprecated = array() ) {
 				$default_gradients = true;
 			}
 			$theme_support_data['settings']['color']['defaultGradients'] = $default_gradients;
+
+			// Classic themes without a theme.json don't support global duotone.
+			$theme_support_data['settings']['color']['defaultDuotone'] = false;
```

Comment:
These is being backported at https://github.com/WordPress/wordpress-develop/pull/2534/files#diff-d6b86476eed058e7cf8b6e57fa52c4fd75b1f0907e1a9ccb0149528a24f7578bR222 If this one lands later, this file can be ignored.